Adding PHP Implementation of Dijkstra's Algo. (Issue #2204)

sukritishah15 opened the pull request:
<!-- Add the issue number that is fixed by this PR (In the form Fixes #45) -->
Fixes #2204 

#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [ ] Input is taken dynamically.
- [x] Sample Input / Output is added at the end of file.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

#### Changes proposed in this pull request:
- 
- 

#### Languages Used:
- PHP 

#### Files Added:
- Dijkstra_Algorithm.php

> We're happy to help you get this ready -- don't be afraid to ask for help.

Thanks!

sukritishah15 commented:
@singh-shreya6 @asha15 @raghav-dalmia @SakshiShreya 
Kindly review my PR for Dijkstra's Algorithm in PHP.

raghav-dalmia commented on Dijkstra_Algorithm/Dijkstra_Algorithm.php:
```diff
@@ -0,0 +1,80 @@
+// Function for Dijkstra'a algorithm
+dijkstra($graph, 4);
+
+
```
remove these extra lines

raghav-dalmia commented on Dijkstra_Algorithm/Dijkstra_Algorithm.php:
```diff
@@ -0,0 +1,80 @@
+3              5
+*/
+
+
```
same here

raghav-dalmia commented on Dijkstra_Algorithm/Dijkstra_Algorithm.php:
```diff
@@ -0,0 +1,80 @@
+}
+
+// Sample Graph (Sample Input)
+$graph = array(
```
Take input dynamically

raghav-dalmia commented on Dijkstra_Algorithm/Dijkstra_Algorithm.php:
```diff
@@ -0,0 +1,80 @@
+    // Print the vertices and their corresponding distances from the source vertex
+    echo "Vertex    Distance from source\n";
+    for($k = 0; $k < $vertices; ++$k){
+        echo $k."              ".$dist[$k]."\n"; 
```
instead of "                    ", use "\t"

raghav-dalmia commented on Dijkstra_Algorithm/Dijkstra_Algorithm.php:
```diff
@@ -0,0 +1,80 @@
+    $visited = array();
+    $dist = array();
+    
+    for ($i = 0; $i < $vertices; ++$i){
```
remove after `for` keyword

raghav-dalmia commented on Dijkstra_Algorithm/Dijkstra_Algorithm.php:
```diff
@@ -0,0 +1,80 @@
+
+/* 
+Sample Input as specified in the code
+
```
add sample input here also

raghav-dalmia commented:
make sure to squash your commits after these changes.

sukritishah15 commented:
@raghav-dalmia I've incorporated the requested changes and squashed the commits. Kindly re-review my PR for implementation of Dijkstra's Algorithm in PHP. Thank you!
